### ISSUE TITLE
Zipline end point issue

### ISSUE THREAD

**Issue by RexTheWho** [simon-wh/PAYDAY-2-BeardLib-Editor#178]
Zipline end points see to reposition themselves after I change the end position and reload.

Edit, Not reset to 0, just reset to whatever location I made their end point be with the buttons that set an end point to where the camera is

**Comment by Luffyyy** [comment]
Can you rephrase this

**Comment by RexTheWho** [comment]
Changing zipline motor end positions in the "translate end position" panel fail to apply.
On reload the end position resets to whatever the position set by the "Use Camera Pos..."
